cmd/telegraf/telegraf.go

```diff
@@ -281,6 +283,8 @@ func runAgent(ctx context.Context,
 		log.Printf("W! Deprecated outputs: %d and %d options", count[0], count[1])
 	}
 
+	_, _ = daemon.SdNotify(false, daemon.SdNotifyReady)
```

Comment:
```suggestion
	// Notify systemd that telegraf is ready
	// SdNotify() only tries to notify if the NOTIFY_SOCKET environment is set, so it's safe to call when systemd isn't present.
	// Ignore the return values here because they're not valid for platforms that don't use systemd.
	// For platforms that use systemd, telegraf doesn't log if the notification failed.
	_, _ = daemon.SdNotify(false, daemon.SdNotifyReady)
```

Let's add a comment here in case there are issues with systemd notification in the future.

Reply:
Sounds fair! Thanks for the comments.

Implemented.